api/src/logger.ts

```diff
@@ -0,0 +1,15 @@
+import {createLogger, format, transports } from 'winston';
+const {combine, colorize, timestamp, prettyPrint } = format;
+
+const defaultLoggingConfig = {
+    format: combine(timestamp(), prettyPrint(), colorize()),
```

Comment:
color is cool, but I don't know how it will show up in kubernetes to be honest so it might not be great.